generators/app/index.js

```diff
@@ -145,6 +159,13 @@ module.exports = class extends BaseGenerator {
         this.skipCheckLengthOfIdentifier = this.configOptions.skipCheckLengthOfIdentifier =
             this.options['skip-check-length-of-identifier'] || this.config.get('skipCheckLengthOfIdentifier');
         this.jhiPrefix = this.configOptions.jhiPrefix = _.camelCase(this.config.get('jhiPrefix') || this.options['jhi-prefix']);
+
+        this.entitySuffix = this.configOptions.entitySuffix =
+            this.config.get('entitySuffix') === false ? '' : this.config.get('entitySuffix') || this.options['entity-suffix'];
+
+        this.dtoSuffix = this.configOptions.dtoSuffix =
```

Comment:
@mselerin where are this being saved to config?

Reply:
Same files, lines 380-381 in the `saveConfig()` function : 
```javascript
            saveConfig() {
                const config = {
                    jhipsterVersion: packagejs.version,
                    applicationType: this.applicationType,
                    baseName: this.baseName,
                    testFrameworks: this.testFrameworks,
                    jhiPrefix: this.jhiPrefix,
                    entitySuffix: this.entitySuffix,
                    dtoSuffix: this.dtoSuffix,
                    skipCheckLengthOfIdentifier: this.skipCheckLengthOfIdentifier,
                    otherModules: this.otherModules,
                    enableTranslation: this.enableTranslation,
                    clientPackageManager: this.clientPackageManager
                };
```

Reply:
Sorry missed it somehow